Dockerfile

```diff
@@ -1,27 +1,31 @@
-FROM node:lts-alpine3.11
```

Comment:
Dockerfile changes breaks using Docker for development. Added #153 as a PR into this one to fix that. Let me know what you think.